packages/lib-classifier/package.json

```diff
@@ -1,4 +1,4 @@
-{
+  {
```

Comment:
Accidental indentation change here?

Reply:
Whoops! I don't know hot this got there, thanks for spotting it.